packages/frontend/src/plugins/authHelpers.js

```diff
@@ -24,37 +27,33 @@ export async function checkAccessCodeAndGetTokens() {
 }
 
 /**
- * Gets the user id and suuid and sets them in local storage.
+ * Gets the user id and suuid, sets them in local storage
+ * @param {import('apollo-client').ApolloClient} apolloClient
  * @return {Object} The full graphql response.
  */
-export async function prefetchUserAndSetSuuid() {
-  let token = localStorage.getItem('AuthToken')
-  if (token) {
-    let testResponse = await fetch('/graphql', {
-      method: 'POST',
-      headers: {
-        Authorization: 'Bearer ' + token,
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify({ query: `{ user { id email suuid streams { totalCount } } }` })
-    })
+export async function prefetchUserAndSetSuuid(apolloClient) {
+  let token = localStorage.getItem(LocalStorageKeys.AuthToken)
+  if (!token) return
 
-    let data = (await testResponse.json()).data
-    if (data.user) {
-      // eslint-disable-next-line camelcase
-      let distinct_id =
-        '@' +
-        crypto.createHash('md5').update(data.user.email.toLowerCase()).digest('hex').toUpperCase()
-
-      localStorage.setItem('suuid', data.user.suuid)
-      localStorage.setItem('distinct_id', distinct_id)
-      localStorage.setItem('uuid', data.user.id)
-      localStorage.setItem('stcount', data.user.streams.totalCount)
-      return data
-    } else {
-      await signOut()
-      throw new Error('Failed to set user')
-    }
+  // Pull user info (& remember it in the Apollo cache)
+  const { data } = await apolloClient.query({
```

Comment:
Moved from `fetch`  to an apollo client query so that we get to keep the results in the Apollo cache